javascript/extractor/lib/typescript/src/virtual_source_root.ts

```diff
@@ -0,0 +1,38 @@
+import * as pathlib from "path";
+import * as ts from "./typescript";
+
+/**
+ * Mapping from the source root to the virtual source root.
```

Comment:
Perhaps say "real source root" here, since that's the terminology used below? Also, would it perhaps be worth briefly outlining what these two roots are and how they relate to each other? (Perhaps something like the comment on line 12 below.)